front/src/Phaser/Game/GameScene.ts

```diff
@@ -575,6 +570,12 @@ export class GameScene extends ResizableScene implements CenterListener {
     }
 
     private triggerOnMapLayerPropertyChange(){
+        this.gameMap.onPropertyChange('exitSceneUrl', (newValue, oldValue) => {
+            if (newValue) this.onMapExit(newValue as string);
+        });
+        this.gameMap.onPropertyChange('exitUrl', (newValue, oldValue) => {
+            if (newValue) this.onMapExit(newValue as string);
+        });
```

Comment:
I don't understand how we can call the same "onMapExit" in exitSceneUrl and in exitUrl.

exitSceneUrl is supposed to contain a path to a map (either absolute or relative to the current map)
exitUrl is supposed to contain a "workadventure" url.

So I ind it surprising you manage to deal with those 2 different URLs in one method.

Reply:
Because I have one specific method that can deal with both cases. And i though exitUrl could also use a relative path.